### ISSUE TITLE
Script link - Remove visually hidden text

### ISSUE THREAD

**Issue by greenc05** [BBC-archive/psammead#3290]
**Describe the bug**
The script link has visually hidden text with english translations on:

https://www.bbc.com/serbian/lat 
https://www.bbc.com/serbian/cyr 
https://www.bbc.com/ukchina/simp 
https://www.bbc.com/ukchina/trad

**To Reproduce**
Steps to reproduce the behaviour:

1. Go to https://www.bbc.com/serbian/cyr
2. Inspect the script link
3. You will see ```<span class="VisuallyHiddenText-sc-1yjwwa5-0 jFfEvX"> Cyrillic</span>```

**Expected behaviour**
The script link should not have visually displayed text as per the UX screen reader spec. This also means that the displayed text should not have an aria hidden attribute, this will also need removing ```<span aria-hidden="true">Ћир</span>```

**Screenshots**
<img width="1220" alt="Screen Shot 2020-03-23 at 11 44 16" src="https://user-images.githubusercontent.com/1061401/77313978-bd32c180-6cfc-11ea-999b-8210e4cfb5b7.png">

**Testing notes**

- [x] This bug fix is expected to need manual testing. When using AT, the script link should just read out the displayed text.

**Comment by sareh** [comment]
@greenc05 So just to clarify:
Before
```
<a href="/serbian/cyr" data-variant="cyr">
  <span>
    <span aria-hidden="true">Ћир</span>
    <span class="VisuallyHiddenText"> Cyrillic</span>
  </span>
</a>
```
After:
```
<a href="/serbian/cyr" data-variant="cyr">Ћир</a>
```

And these changes should be seen on serbian lat/cyr, and ukchina and zhongwen simp/trad too.

**Comment by sareh** [comment]
The Psammead component needs to have its Readme updated, Storybook file and Test file updated as well as not surrounding the index in a `<span></span>` Then in Simorgh we can make sure we are not passing in the visually hidden text component.
